docs/.gitignore

```diff
@@ -0,0 +1,4 @@
+.vagrant
+node_modules/*
+09-writing-advanced-plans/modules/stdlib
+_site
```

Comment:
Pretty sure this is the only line that's needed out of this file.

Reply:
Updated.